components/blog-post-with-image.js

```diff
@@ -1,13 +1,18 @@
 import Link from 'next/link';
 import { ImageTextWrapper } from './image-text-wrapper';
+import { ReadingMinutes } from './reading-minutes';
 import { RichText } from './rich-text';
 
-export const BlogPostWithImage = ({ blog: { title, banner, summary }, path }) => {
+export const BlogPostWithImage = ({
+  blog: { title, banner, summary, blocks },
+  path,
+}) => {
   return (
```

Comment:
Here you can create a hook to get the reading minutes from blocks so you can reuse same hook in other places. For instance:
```javascript
//use-reading-minutes-from-blocks.js
export const useReadingMinutesFromBlocks = (blocks) => {
    const allTheTextToBeCalculated = blocks.filter(block => block.template === 'text-block').reduce((cumulatedText, block) => `${cumulatedText}${block.title}${block.content}`, '');
  const readingMinutes = '' // here calculate the reading minutes based on allTheTextToBeCalculated
  return readingMinutes;
}


//BlogPostWithImage
export const BlogPostWithImage = ({
  blog: { title, banner, summary, blocks },
  path,
}) => {
  const readingMinutes = useReadingMinutesFromBlocks(blocks);
  return (
```